plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/drivers/WebDriverService.java

```diff
@@ -179,7 +179,14 @@ private void terminateProcessIE() {
             Runtime.getRuntime().exec("taskkill /f /im iexplore.exe").waitFor();
             LOG.info("All iexplorer processes were terminated");
         } catch (IOException | InterruptedException e) {
-            LOG.warn("Failed to wait for browser processes finish", e);
+            LOG.info("Failed to wait for browser processes finish", e);
+        }
+        try {
+            LOG.info("Trying to terminate iexplorer processes");
+            Runtime.getRuntime().exec("taskkill /f /im iexplore.exe").waitFor();
+            LOG.info("All iexplorer processes were terminated");
+        } catch (IOException | InterruptedException e) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either re-interrupt this method or rethrow the "InterruptedException". ([squid:S2142](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS2142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=61&issues=AWRpLRJ8Mt9UOhrw_2_s&open=AWRpLRJ8Mt9UOhrw_2_s)

[ISSUE_KEY]: AWRpLRJ8Mt9UOhrw_2_s